components/pipelines-as-code/base/kustomization.yml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://github.com/konflux-ci/konflux-ci/dependencies/pipelines-as-code?ref=fa491580946a7ae5feac06d6a345248f1787a313
+
+
+patches:
```

Comment:
The kustomization file we refer to already contains the patch (and other patches).
We can remove it form this file